administering_a_cluster/connected-to-diconnected.adoc

```diff
@@ -0,0 +1,16 @@
+[id="connected-to-diconnected"]
+= Converting a connected cluster to a disconnected cluster
+include::modules/common-attributes.adoc[]
+:context: connected-to-diconnected
+
+toc::[]
+
+
+
+
+include::modules/connected-to-diconnected-prepare-mirror.adoc[leveloffset=+1]
+include::modules/connected-to-diconnected-config-registry.adoc[leveloffset=+1]
+include::modules/connected-to-diconnected-move-apps.adoc[leveloffset=+1]
```

Comment:
I can't find this file in the repo or the PR.

Reply:
Good catch. I believe I first added the assembly to the _Administering a cluster_ book before realizing it is only in Dedicated. I removed this file.